octodns/record/__init__.py

```diff
@@ -1067,15 +1067,18 @@ class _ChunkedValuesMixin(_ValuesMixin):
     CHUNK_SIZE = 255
     _unescaped_semicolon_re = re.compile(r'\w;')
 
+    def chunked_value(self, value):
+        value = value.replace('"', '\\"')
+        vs = [value[i:i + self.CHUNK_SIZE]
+              for i in range(0, len(value), self.CHUNK_SIZE)]
+        vs = '" "'.join(vs)
+        return '"{}"'.format(vs)
```

Comment:
This logic needed to be broken out of the values version below so that it could be called on individual values by _Route53Record. More on this below.